openedx/core/lib/block_structure/tests/test_block_structure.py

```diff
@@ -131,6 +131,7 @@ def test_xblock_data(self):
         block_structure = BlockStructureModulestoreData(root_block_usage_key=0)
         for block in blocks:
             block_structure._add_xblock(block.location, block)
+            block_structure._get_or_create_block(block.location)
```

Comment:
**Note to reviewers:** Another test issue that was surfaced where it incorrectly formed the block_structure and ended up acting on `None` blocks.